### DISCUSSION TITLE
Unable to connect external zookeeper cluster

### DISCUSSION THREAD

**Discussion by rajat965ng** [strimzi/strimzi-kafka-operator#6662] [Q&A]
Although it is mentioned in the documentation that "zookeeper.connect" is a forbidden option in the config property. But I have a use case where I have an additional zookeeper cluster present outside the Kubernetes cluster. Let me know 
1. How to make Strimzi provisioned zookeeper as part of external zookeeper?
2. How to make Kafka connect to an external zookeeper cluster?

**Comment by scholzj** [comment]
This is not supported by Strimzi. Strimzi deployed Kafka cluster always uses its own ZooKeeper.
